src/test/java/io/vertx/openapi/test/E2ETest.java

```diff
@@ -86,6 +88,38 @@ void testExtractPath(String basePath, VertxTestContext testContext) {
     }).onFailure(testContext::failNow);
   }
 
+  //These types come from: https://spec.openapis.org/oas/latest.html#special-considerations-for-multipart-content
+  // 1 primitive "petId"
+  // 1 complex "json"
+  // 1 primitive with content encoding aka png base64 encoded.
+  @Test
+  @Timeout(value = 2, timeUnit = TimeUnit.SECONDS)
+  @DisplayName("Send a multipart/form-data request")
+  public void sendMultipartFormDataRequest(VertxTestContext testContext) {
+    Path path = getRelatedTestResourcePath(E2ETest.class).resolve("multipart.txt");
+
```

Comment:
Are these two empty lines intended?

Reply:
It'll be taken up now by the expected petmeta data JsonObject.